spreadsheet/chapter.md

```diff
@@ -256,7 +256,7 @@ If it is, we use `$timeout` to schedule a change of cell focus after the current
 
 The `const` declarator means `direction` will not change during the function’s execution. The direction to move is either upward (`-1`, from **A2** to **A1**) if the key code is 38 (**UP**), or downward (`+1`, from **A2** to **A3**) otherwise.
 
-Next up, we retrieve the target element using the ID selector syntax (e.g. `"#A3"`), constructed with a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings) written in a pair of back-quotes, concatenating the leading `#`, the current `col` and the target `row + direction`:
+Next up, we retrieve the target element using the ID selector syntax (e.g. `"#A3"`), constructed with a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings) written in a pair of backticks, concatenating the leading `#`, the current `col` and the target `row + direction`:
```

Comment:
"backtick" seems to be the most commonly used word for this character these days. 

https://books.google.com/ngrams/graph?content=back+quote%2Cbacktick&year_start=1960&year_end=2015&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cback%20quote%3B%2Cc0%3B.t1%3B%2Cbacktick%3B%2Cc0